.github/workflows/test-bug-run-badge.yml

```diff
@@ -0,0 +1,78 @@
+name: Test new bug report badge
+run-name: Test bug report on issue ${{ github.event.issue.number }}
+on:
+  issues:
+    types: [opened]
+jobs:
+  extract-bug-badge-url:
+    if: ${{ contains(github.event.issue.labels.*.name, 'question') }}
+    runs-on: ubuntu-latest
+    outputs:
+      runBadgeTest: ${{ steps.testCondition.outputs.runNext }}
+      link: ${{ steps.testCondition.outputs.link }}
+    steps:
+      - name: Test badge test run conditions
+        id: testCondition
+        run: |
+          product=$(echo "${{ github.event.issue.body }}" | grep -A2 "Are you experiencing an issue with.*" | tail -n 1)
+          link=$(echo "${{ github.event.issue.body }}" | grep -A2 "Link to the badge.*" | tail -n 1)
+
+          if [[ "$product" == "shields.io" && "$link" == "https://img.shields.io"* ]]; then
+            echo "runNext=true" >> "$GITHUB_OUTPUT"
+            echo "link=$link" >> "$GITHUB_OUTPUT"
+          else
+            echo "Conditions not met. Skipping the workflow..."
+            echo "runNext=false" >> "$GITHUB_OUTPUT"
+          fi
+
+  run-bug-badge-url-test:
+    needs: extract-bug-badge-url
+    if: needs.extract-bug-badge-url.outputs.runBadgeTest == 'true'
+    permissions:
+      issues: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup
+        uses: ./.github/actions/setup
+        with:
+          node-version: 16
```

Comment:
For the moment, this needs to stay as 16 due to `strict-englines`, but I'm going to quickly note here that when we merge https://github.com/badges/shields/pull/9385 we will need to make this 18.

Reply:
cool, whatever gets merged first should update the later

Reply:
:+1: I'll update https://github.com/badges/shields/pull/9385 after I merge this